content/blog/exposing-an-application-using-ingress-and-tls-termination-on-kubernetes-in-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,528 @@
+Name:             ingress-host-based-selfsigned
+Labels:           <none>
+Namespace:        nginx-apps
+Address:
+Ingress Class:    nginx
+Default backend:  <default>
+TLS:
+  cfe-tls-key-pair terminates nginx.example.com
+Rules:
+  Host                     Path  Backends
+  ----                     ----  --------
+  nginx.example.com
+                           /   nginx-main:80 (10.192.4.44:80)
+  blue.nginx.example.com
+                           /   nginx-blue:80 (10.192.3.78:80)
+  green.nginx.example.com
+                           /   nginx-green:80 (10.192.4.45:80)
+Annotations:               cert-manager.io/issuer: nginx-selfsinged-issuer
+                           ingress.kubernetes.io/ssl-redirect: true
+Events:
+  Type    Reason             Age   From                       Message
+  ----    ------             ----  ----                       -------
+  Normal  Sync               20s   nginx-ingress-controller   Scheduled for sync
+  Normal  CreateCertificate  20s   cert-manager-ingress-shim  Successfully created Certificate "cfe-tls-key-pair"
+```
+
+### Access deployed Nginx applications
+
+B﻿efore start accessing the deployed Nginx applications, you need set up the domain and the subdomain name resolution. For the sample domain name *nginx.example.com*, and its subdomains, *blue.nginx.example.com* and *green.nginx.example.com*, the workstation host file has been used for DNS resolution.  
+
+Type the following commands to check that this is done correctly: 
+
+```shell
+$ host nginx.example.com
+nginx.example.com has address 10.6.115.251
+
+$ host green.nginx.example.com
+green.nginx.example.com has address 10.6.115.251
+
+$ host blue.nginx.example.com
+blue.nginx.example.com has address 10.6.115.251
+```
+
+You can then validate the Ingres TLS configuration of the deployed Nginx applications using the browser. 
+
+S﻿tart the browser and type the URL *nginx.example.com*, it will be rediected over HTTPS with the warning message *'Your connection is not private'*: 
+
+![](/img/nginx-main-warning.png)
+
+T﻿his is due to the fact that the self-signed certifcate is generated in cert-manager and configured in the K8s Ingress resource.
+
+C﻿lick *Not secure* and start the Certificate Viewer to check the certificate:
+
+![](/img/nginx-main-cert.png)
+
+C﻿lick *Proceed to nginx.example.com (unsafe)*, you then go to the Nginx *MAIN* page:
+
+![](/img/nginx-main.png)
+
+Type the URL *green.nginx.example.com* to the browser, it will be rediected over HTTPS with the same warning message *'Your connection is not private'*:   
+
+![](/img/nginx-green-warning.png)
+
+C﻿lick *Proceed to green.nginx.example.com (unsafe)*, you then go to the Nginx *GREEN* page:
+
+![](/img/nginx-green.png)
+
+T﻿he same thing occurs when type the URL *blue.nginx.example.com* to the browser. The access will be rediected over HTTPS with the same warning message *'Your connection is not private'*:  
```

Comment:
Typo: The same thing occurs when you type the URL...

Typo: The access will be redirected over HTTPS with....